setup.py

```diff
@@ -39,7 +39,7 @@
         #
         # As a mitigation, run `pip uninstall allennlp` before installing this
         # package.
-        sha = "88fe0075c28babb20f076c43f932b77d80ce81a3"
+        sha = "28028a0544632307fe48e66a15a4b2a22c3dddda"
```

Comment:
Heads up, for the reasons outlined in the warning above, the "Release" build (http://build.allennlp.org/viewType.html?buildTypeId=AllenNLPHub_Release) will not be meaningful unless all the sub-repos use the same SHA (or use actual released versions of AllenNLP -- which can then be different as `pip` should identify conflicts). Just wanted to mentioned this in case when you unblock the other hub tests the "Release" build stays broken.

Reply:
I'm trying to get the PRs to test successfully first.

Reply:
👍